common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -643,6 +644,7 @@ def set_logged_in_cookies(backend=None, user=None, strategy=None, auth_entry=Non
                     pass
                 else:
                     response = redirect(redirect_url)
+                    set_logged_in_edly_cookies(request, response, user)
                     return user_authn_cookies.set_logged_in_cookies(request, response, user)
```

Comment:
@rehan99000 I don't think we should call our method `set_logged_in_edly_cookies` everywhere we are setting cookies instead we should call it only inside `set_logged_in_cookies` of user_authn app here `openedx.core.djangoapps.user_authn.cookies.set_logged_in_cookies`

Reply:
That's a good idea. let me make this change.